src/spaces.ts

```diff
@@ -11,6 +11,20 @@ const requireSkin = require.context(
 
 requireSkin.keys().map(file => requireSkin(file));
 
+type Space = {
```

Comment:
This format will change a lot, we may not want to maintain the structure here for now.